brave/src/test/java/com/linecorp/armeria/server/brave/BraveServiceTest.java

```diff
@@ -148,7 +148,7 @@ void scopeDecorator() throws Exception {
 
         // check service name
         assertThat(span.localServiceName()).isEqualTo(TEST_SERVICE);
-        assertThat(scopeDecoratorCallingCounter.get()).isEqualTo(3);
+        assertThat(scopeDecoratorCallingCounter.get()).isEqualTo(1);
```

Comment:
I guess neither 3 nor 1 mean much to me. We probably should add a comment explaining what this is actually asserting semantically.

Reply:
Or we can remove if it's too strongly tied to implementation details to be helpful.

Reply:
yeah good point.. the reduction by two is not scoping during parsing :) I'll make a comment that the presence of 1 is for downstream handlers. usually I check some interceptor can read the context vs counting, though in this case counting was a good thing as it shows the better performance we should expect

Reply:
Maybe `.isGreaterThanZero()` then?

Reply:
added isOne as incrementing would be a bug unless intentional